DatadogRUM/Sources/Feature/RUMFeature.swift

```diff
@@ -97,6 +97,8 @@ internal final class RUMFeature: DatadogRemoteFeature {
                     return nil
                 }
             }(),
+            renderLoopObserver: DisplayLinker(notificationCenter: .default),
```

Comment:
blocking/ We must use [`configuration.notificationCenter`](https://github.com/DataDog/dd-sdk-ios/blob/5cc5fcd31c1271d3aef92cd8395152ac87dcc836/DatadogRUM/Sources/RUMConfiguration.swift#L316-L317) for dependency injection in integration tests:
```suggestion
            renderLoopObserver: DisplayLinker(notificationCenter: configuration.notificationCenter),
```

This way, when adding integration coverage we can control this `notificationCenter`:
```swift
var config: RUM.Configuration = .mockAny()
config.notificationCenter = custom

RUM.enable(with: config, in: core)
```

Reply:
Perfect. ty